README.md

```diff
@@ -53,10 +66,14 @@ python -m myosuite.utils.examine_env --env_name myoElbowPose1D6MRandom-v0
 mjpython -m myosuite.utils.examine_env --env_name myoElbowPose1D6MRandom-v0
 ```
 
+
+
 ## Examples
 It is possible to create and interface with MyoSuite environments just like any other OpenAI gym environments. For example, to use the `myoElbowPose1D6MRandom-v0` environment, it is possible simply to run: [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1zFuNLsrmx42vT4oV8RbnEWtkSJ1xajEo)
 
 
+ <!-- TODO: consider instructions to gym api or introduce more annotation for the code each line -->
```

Comment:
@raku-slyu -- what's the plan here?

Reply:
Comments on introducing gym api has been added on line 36 so removed this comment